wled00/data/settings_leds.htm

```diff
@@ -23,6 +23,20 @@
 		function isD2P(t) { return t > 47 && t < 64; }                      // is digital 2 pin type
 		function is16b(t) { return t == 26 || t == 29 }                     // is digital 16 bit type
 		function isVir(t) { return t >= 80 && t < 96; }                     // is virtual type
+		function isHub75(t) { return t >= 100 && t <= 110; }                // is Hub75 type
+		function hideHub75() {
```

Comment:
I would prefer the other way around. Instead of hiding Hub75 types, insert them into SELECT.
It will require  more effort in xml.cpp and some modified code here but should allow for enhancements in the future.

I will tackle this problem and push modifications when I'm done and the solution is viable. No need for you to do anything.